src/components/Sections/SectionChart/SectionChart.less

```diff
@@ -3,13 +3,7 @@
 .section-chart {
   height: 100%;
   width: 100%;
-  display: table;
   .section-title {
     font-size: 18px;
-    margin: 10px 0;
-    display: table-row;
-  }
-  .section-chart-content {
-    display: table-row;
   }
```

Comment:
this causes the bar chart to be outside the widget border 
![image](https://user-images.githubusercontent.com/61918543/116007365-2b406b00-a618-11eb-8d84-e8e6d88a77d5.png)


before fix
![image](https://user-images.githubusercontent.com/61918543/116007329-f3392800-a617-11eb-8956-e5c51a1ba5b2.png)